draft-ietf-quic-qpack.md

```diff
@@ -215,8 +215,7 @@ and follows the definitions in [RFC7541] without modification.
 Entries in the QPACK static and dynamic tables are addressed separately.
 
 Entries in the static table have the same indices at all times.  The static
-table is defined in Appendix A of {{!RFC7541}}. Note that because HPACK did not
-use zero-based references, there is no value at index zero of the static table.
+table is defined in {{static-table}}.
```

Comment:
I think {{static-table}} might be missing...

Reply:
Implicit anchor on the title of the section; turns into "Appendix A" in the rendered document.